assembl/tasks/translate.py

```diff
@@ -14,7 +14,26 @@ def languages_for(self, locale_code, db=None):
         return ()
 
 
+class LanguagesTranslationTable(TranslationTable):
```

Comment:
Sorry I did not think about it initially, this could have been done with a PrefCollectionTranslationTable using a LanguagePreferenceCollectionWithDefault... I don't think it's worth changing now. And your version handles multiple targets, which is better.

Reply:
Thanks, I'll merge then